packages/dd-trace/src/plugins/index.js

```diff
@@ -38,6 +38,7 @@ module.exports = {
   'paperplane': require('../../../datadog-plugin-paperplane/src'),
   'pg': require('../../../datadog-plugin-pg/src'),
   'pino': require('../../../datadog-plugin-pino/src'),
+  'pino-pretty': require('../../../datadog-plugin-pino-pretty/src'),
```

Comment:
Do you think we should have pino-pretty be standalone or directly inside the pino plugin ?

Reply:
I think it should be inside the pino plugin. I don't feel strongly about this though. @rochdev ?

Reply:
I agree. The idea for plugins is that is does what a user would expect when enabled. When enabling the `pino` plugin, I would expect `pino` to just work without having to explicitly enable individual formatters etc, so in that sense it makes more sense to have a single plugin.

Reply:
Alright, make sense, I'll merge it back to the pino plugin.